nightingale_navigation/src/room_runner.py

```diff
@@ -147,6 +191,14 @@ def fb_cb(fb):
                 [fb.base_position.pose.position.x, fb.base_position.pose.position.y]
             )
             euclidean_distance = np.linalg.norm(target - cur)
+            if euclidean_distance < self.deflation_hysteresis_low:
```

Comment:
I'm not sure this line guarantees hysteresis. Is hysteresis the desired behaviour? It seems like this method might work fine

Reply:
im not sure what u mean?

Reply:
yes, hysteresis is desired